guides/common/modules/con_ipv4-and-ipv6-requirements.adoc

```diff
@@ -0,0 +1,44 @@
+[id="ipv4-and-ipv6-requirements_{context}"]
+= IPv4 and IPv6 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+
+ifdef::satellite[]
+[NOTE]
+====
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+====
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure that an IPv6 loopback is configured.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+ifdef::satellite[]
++
+For a supported way to disable the IPv6 protocol, see link:https://access.redhat.com/solutions/5045841[How do I disable the IPv6 protocol on Red Hat Satellite and/or Red Hat Capsule server?] in Red{nbsp}Hat Knowledgebase.
+endif::[]
+
+The following requirements apply to installations in an IPv6 network:
+
+* You must deploy an external DHCP IPv6 server as a separate, unmanaged service to bootstrap clients into GRUB2.
```

Comment:
This is only true if you provision. If you don't there's no requirement to deploy this.